### PR TITLE
Create new post

### PR DESCRIPTION
**Pull request by b0llu**
## What to check
- [ ] src/Context/Post.context.js
- [ ] src/Context/Reducer.context.js

## What not to check
- [ ] Do Not Check Any Other Files

live Preview,
https://stormmedia.netlify.app/

### CODE REVIEW

**Review by mikesch803 — APPROVED** [review]
I liked the design that you have replicated, the code is clean, good work!

File: src/App.js [hunk 1]
```diff
@@ -8,7 +8,9 @@ import {
   RightSidebar,
   ThemeToggler,
 } from "Components";
+import MockAPI from "Mockman";
 import { Routes, Route, useLocation } from "react-router-dom";
+import { ToastContainer } from "react-toastify";
 import { HomePage, LandingPage } from "./Pages";
```
Comment (Dev-AkshatGupta) [inline src/App.js:14]: I would suggest you to use absolute imports much better and easy to understand

File: src/Components/EachPost/EachPost.jsx [hunk 2]
```diff
@@ -1,24 +1,49 @@
...
-          src="https://pbs.twimg.com/profile_images/1464407388228780036/NFY5UUPn_bigger.jpg"
+          src={
+            post.profilePhoto
+              ? post.profilePhoto
+              : "https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png"
+          }
           alt="User Thumbnail"
         />
         <div className={styles.post_user_details}>
           <div className={styles.user_name}>
```
Comment (Dev-AkshatGupta) [inline src/Components/EachPost/EachPost.jsx:35]: Nice that you are using css modules impressed by this approach

**Review by Dev-AkshatGupta — APPROVED** [review]
Amazing work Dhruv 
Given a suggestion nothing specific just a advice will help you and you are good to merge this

File: src/App.js (hunk 1 above)
Comment (b0llu) [reply src/App.js:14]: I am using absolute import this was left out by mistake

**Pull request merged**